src/sci/impl/protocols.cljc

```diff
@@ -177,14 +176,34 @@
                              (map #(process-single fq %) fn-body))
                            :else fn-body)]
          (if default-method?
-           `(defmethod ~fq
-              :default
-              ~@fn-body)
+           #?(:clj
+              `(defmethod ~fq
+                :default
+                ~@fn-body)
+              :cljs
+              `(do
+                 (defmethod ~fq
+                   :default
+                   ~@fn-body)
+                 (defmethod ~fq
+                   ~type
+                   ~@fn-body)))
```

Comment:
This is kind of a hacky way of doing this. Basically, I define the `:default` which will be hit any time someone calls the protocol method on a type, and _also_ defines a `::extend-default` method which it uses later as a signal for `satisfies?` to return true.